lntest/itest/lnd_test.go

```diff
@@ -9423,6 +9427,10 @@ func assertDLPExecuted(net *lntest.NetworkHarness, t *harnessTest,
 	dave *lntest.HarnessNode, daveStartingBalance int64,
 	anchors bool) {
 
+	// Increase the fee estimate so that the following force close tx will
+	// be cpfp'ed.
+	net.SetFeeEstimate(30000)
```

Comment:
we could inspect the mempool txs and check that they indeed have the feerate we want.

Reply:
added